src/app/utils/ThemeUtils.ts

```diff
@@ -268,15 +270,17 @@ export default class ThemeUtils {
 		if (snow) {
 			try {
 				snow.destroy()
-			} catch (error) {}
+			} catch (error) {
+				console.warn('error in stopSnow()', error)
+			}
 		}
 	}
 
-	private shadeColor(color_: string, percent: number): string {
+	/*private shadeColor(color_: string, percent: number): string {
```

Comment:
As mentioned, I would prefer to remove whole functions that are commented out. It's a matter of taste though.

Reply:
Removed